azure-pipelines.yml

```diff
@@ -403,8 +403,11 @@ stages:
           arguments: -test -configuration Release
         condition: or(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['compilerChange'], 'true'))
 
-      - script: $(Build.SourcesDirectory)\.dotnet\dotnet.exe tool run dotnet-format whitespace  $(Build.SourcesDirectory)\src --folder --include-generated --include $(Build.SourcesDirectory)\src\Compilers\CSharp\Portable\Generated\ $(Build.SourcesDirectory)\src\Compilers\VisualBasic\Portable\Generated\ $(Build.SourcesDirectory)\src\ExpressionEvaluator\VisualBasic\Source\ResultProvider\Generated\ --verify-no-changes
+      - task: PowerShell@2
```

Comment:
Curious: why did you pick the PowerShell task here vs using `pwsh`? That is installed in our repo as a local tool. Generally I try and use that when possible.

Reply:
Will do.